atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/CassandraCliParser.java

```diff
@@ -41,7 +41,7 @@ public int parseSystemAuthReplicationFromCqlsh(String output) throws IllegalArgu
                 }
             }
         } catch (Exception e) {
-            log.error("Failed parsing system_auth keyspace RF: " + e);
+            log.error("Failed parsing system_auth keyspace RF.", e);
```

Comment:
I'm not sure whether we should put periods on the end of these. If we want them though, there's a change below that doesn't add it. I personally would vote for no periods.